backend/api/views/tests/test_statistics_r.py

```diff
@@ -263,9 +225,9 @@ def test_get_total(client):
         )
 
     resp = client.get("/api/statistics/total")
-    assert resp.json()["臺南市"]["documents"] == 9
+    assert resp.json()["臺南市"]["documents"] == expected
     count = resp.json()["臺南市"][DocumentDisplayStatusConst.IN_PROGRESS]
-    assert count == 9, f"expect 9 but {count}"
+    assert count == expected, f"expect {expected} but {count}"
```

Comment:
There are some places like this which could be refactored to be more flexible and solid.